[2025-07-07T20:32:03.673+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: axone_fb_etl_pipeline.extract manual__2025-07-07T20:26:54.288448+00:00 [queued]>
[2025-07-07T20:32:03.691+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: axone_fb_etl_pipeline.extract manual__2025-07-07T20:26:54.288448+00:00 [queued]>
[2025-07-07T20:32:03.692+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 4
[2025-07-07T20:32:03.722+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): extract> on 2025-07-07 20:26:54.288448+00:00
[2025-07-07T20:32:03.769+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2025-07-07T20:32:03.776+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'axone_fb_etl_pipeline', 'extract', 'manual__2025-07-07T20:26:54.288448+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpw8ehonkb']
[2025-07-07T20:32:03.782+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask extract
[2025-07-07T20:32:03.963+0000] {task_command.py:415} INFO - Running <TaskInstance: axone_fb_etl_pipeline.extract manual__2025-07-07T20:26:54.288448+00:00 [running]> on host d19281c80524
[2025-07-07T20:32:04.259+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='axone_fb_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T20:26:54.288448+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T20:26:54.288448+00:00'
[2025-07-07T20:32:04.264+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-07T20:32:04.266+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/*** && python src/extract.py --query "deces jacques chirac" --num_scrolls 10']
[2025-07-07T20:32:04.281+0000] {subprocess.py:86} INFO - Output:
[2025-07-07T20:32:04.908+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-07-07T20:32:04.910+0000] {subprocess.py:93} INFO -   File "src/extract.py", line 8, in <module>
[2025-07-07T20:32:04.910+0000] {subprocess.py:93} INFO -     from selenium import webdriver
[2025-07-07T20:32:04.910+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'selenium'
[2025-07-07T20:32:04.938+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-07-07T20:32:04.959+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-07-07T20:32:04.967+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=axone_fb_etl_pipeline, task_id=extract, execution_date=20250707T202654, start_date=20250707T203203, end_date=20250707T203204
[2025-07-07T20:32:04.984+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 5 for task extract (Bash command failed. The command returned a non-zero exit code 1.; 236)
[2025-07-07T20:32:05.012+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-07-07T20:32:05.054+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
